cmd/tofu/main.go

```diff
@@ -117,6 +114,7 @@ func realMain() int {
 	streams, err := terminal.Init()
 	if err != nil {
 		Ui.Error(fmt.Sprintf("Failed to configure the terminal: %s", err))
+		tracing.ForceFlush(5 * time.Second)
```

Comment:
What's so special about the ForceFlush and why don't we do it in a defer call as it's part of all the error paths?

Reply:
Because I wrote this early in the day and forgot that defer is a thing 🤦 please forgive me! I'll re-work this.